components/Transactions/CoinPicker.tsx

```diff
@@ -6,6 +6,9 @@ import {
   ITransactionsContext,
 } from "../../providers/TransactionsProvider";
 
+import {Box, Text} from "native-base";
```

Comment:
Can we move this import statement up to be w/ everything that's not our modules? Like between line3/4.